runtime/browser/ime_application.cc

```diff
@@ -25,8 +25,9 @@
 namespace runtime {
 
```

Comment:
IME Application did't support Notification Window, 
So, didn't change this class

Is there a reason for change?